workflow/controller/dag.go

```diff
@@ -382,6 +382,8 @@ func (woc *wfOperationCtx) executeDAGTask(ctx context.Context, dagCtx *dagContex
 		scope, err := woc.buildLocalScopeFromTask(dagCtx, task)
 		if err != nil {
 			woc.markNodeError(node.Name, err)
+			woc.log.Errorf("Got a nil pointer for scope due to error %s", err)
```

Comment:
@isubasinghe Can you reword the message? "Failed to build local scope from task" and add task name as field

Reply:
@sarabala1979 done